templates/Manage.html

```diff
@@ -3,29 +3,34 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/bootstrap/bootstrap.min.css">
+    <link rel="stylesheet" type="text/css" href="/css/baseStyle.css">
+
 
-    <style>
-        table {font-family: arial, sans-serif;border-collapse: collapse; width: 70%;}
-        td, th {border: 1px solid #dddddd;text-align: left; padding: 8px;}
-        tr:nth-child(even) {background-color: #dddddd;}
-    </style>
-  
     <title>Connexion</title>
 </head>
 <body>
-    <h1>Connex.us</h1>
-    {% if user %}
-        You're signed in as {{ user }} | <a href="{{ login_url }}"> {{ login_text }}</a> 
-        {% if isAdmin %} | <a href="/admin/dashboard">Admin Menu</a> {% endif %} <br> 
-    {% endif %}
-    <hr>
-    <a href="manage">Manage</a> |
-    <a href="create">Create</a> |
-    <a href="view">View</a> |
-    <a href="search">Search</a> |
-    <a href="trending">Trending</a> |
-    <a href="social">Social</a>
-    <p>
+    <div class="container">
+
+        <header>
+           <h1>Connex.us</h1>
+        </header>
+
+        {% if user %}
+            You're signed in as {{ user }} | <a href="{{ login_url }}"> {{ login_text }}</a>
+            {% if isAdmin %} | <a href="/admin/dashboard">Admin Menu</a> {% endif %} <br>
+        {% endif %}
+
+        <hr>
+
+        <div class="topnav">
```

Comment:
This looks cool! Is there a template way to save this HTML snippet to a separate file? Like, so then we can just insert an `include` statement or something in every page? Then we can really easily change the appearance across the whole site!